makefile

```diff
@@ -9,12 +9,21 @@ else
  A=$(ARMOPT)
 endif
 
-LVM=clang
-GNU=gcc-10
+# this could be an extra ./configure step
+# - require either gcc-10 or clang
+ifeq      ($(shell gcc   -v 2>&1|grep -o 'gcc version 10'),gcc version 10)
+ CC=gcc
+ W=-fmax-errors=1 -flax-vector-conversions
+else ifeq ($(shell clang -v 2>&1|grep -o 'clang' ),clang)
```

Comment:
should be `grep -o 'clang version'` otherwise will match `clang: command not found` error